### ISSUE TITLE
peerDependencies with jQuery 3.4

### ISSUE THREAD

**Issue by marcwieland95** [jquery-validation/jquery-validation#2272]
jQuery Validation should also work with the latest jQuery version 3.4, right?

Can we lift the peerDependencies to that version?
NPM audit throws an error because jQuery had some vulnerability prior to that version.

```
"peerDependencies": {
	"jquery": "^1.7 || ^2.0 || ^3.4"
},
```

**Comment by ericfowler303** [comment]
this has been done in this pull: https://github.com/jquery-validation/jquery-validation/pull/2248

This updated code still hasn't been published to the NPM public version so until that's done we'll all keep seeing that error

**Comment by marcwieland95** [comment]
Oh man. This project doesn't look that active. But thanks for the pull-request.

I solved it in the meanwhile with this addition in my `package.json`:
```
"resolutions": {
    "jquery-validation/jquery": "3.4"
}
```

**Comment by staabm** [comment]
fixed with #2248

**Comment by staabm** [comment]
I just published jquery-validateion 1.19.1 to npm.

most important change is that jquery is now defined as a peer dependency, so we dont get into your way. see https://github.com/jquery-validation/jquery-validation/pull/2248

sorry this took us so long. jq-validation is not top priority atm
